.gitignore

```diff
@@ -30,3 +30,6 @@ vendor/
 borders-for-default-menu.json
 defaults.json
 no-toggle-border-fix.json
+
+# Ignore react app autogenerated asset file.
+admin/react/assets/build/dashboard-app.asset.php
```

Comment:
Why not exclude complete build directory?

Reply:
Then should it be main [application JS](https://github.com/brainstormforce/custom-fonts/blob/new-admin-dashboard/admin/react/assets/src/DashboardApp.js) file we need to enqueue [here](https://github.com/brainstormforce/custom-fonts/blob/new-admin-dashboard/admin/react/includes/class-bsf-custom-fonts-menu.php#L238) ?

Reply:
Excluded build and prepared only at time of deploy PS: https://github.com/brainstormforce/custom-fonts/blob/new-admin-dashboard/.github/workflows/push-to-deploy.yml

As this has single react application inside, not made changes with `yarn` make it using `npm` only for now, is that fine?